pyramid_oereb/lib/readers/extract.py

```diff
@@ -129,11 +129,7 @@ def read(self, params, real_estate, municipality):
                 # Filter topics due to topics parameter
                 if not params.skip_topic(plr.theme.code):
                     if isinstance(plr, PlrRecord):
-                        contained = False
-                        for theme in concerned_themes:
-                            if theme.code == plr.theme.code:
-                                contained = True
-                        if not contained:
+                        if plr.theme.code not in [theme.code for theme in concerned_themes]:
```

Comment:
This should not change the logic at all and is not directly linked to this PR, but I struggled to understand this piece of code so I re-wrote this in a way which hopefully is more understandable for others as well